packages/manager/apps/catalog-revamp/src/hooks/useCatalog.ts

```diff
@@ -0,0 +1,73 @@
+import { useState, useEffect } from 'react';
+import { useQuery, UseQueryResult } from '@tanstack/react-query';
+import {
+  getManagerHubCatalogList,
+  getManagerHubCatalogListQueryKey,
+} from '@/api';
+import {
+  groupItems,
+  filterByEnum,
+  getAvailableCategories,
+  matchCategories,
+  matchUniverses,
+  matchSearchText,
+  filterItems,
+  getUniverses,
+} from '@/utils/utils';
+
+export type Product = {
+  id: number;
+  name: string;
+  description: string;
+  lang: string;
+  categories: string[];
+  url: string;
+  regionTags: string[];
+  productName: string;
+  order: string;
+  universe: string;
+  category: string;
```

Comment:
**nitpick:** Category can be evaluated from the `categories` array. It is just added to support AngularJS Stack. So may be we can remove it.

Reply:
category is inside the response of the API request /hub/catalog so we need to keep it to match the API response structure